charts/flyte-core/templates/propeller/service.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if and .Values.flytepropeller.enabled .Values.flytepropeller.service.enabled }}
```

Comment:
Should this not be only enabled when serviceMonitor is enabled?

Reply:
You can still have a service without servicemonitor so there is no harm in creating just the service. This keeps it flexible to have the service for any other ports which may not be related to prometheus port.

Reply:
👍